crates/bevy_render/src/texture/dds.rs

```diff
@@ -18,14 +20,29 @@ pub fn dds_buffer_to_image(
         )));
     }
     let mut image = Image::default();
+    let is_cubemap = dds.header.caps2.contains(Caps2::CUBEMAP);
+    let mut depth_or_array_layers = if dds.get_num_array_layers() > 1 {
+        dds.get_num_array_layers()
+    } else {
+        dds.get_depth()
+    };
+    if is_cubemap {
+        if !dds.header.caps2.contains(
+            Caps2::CUBEMAP_NEGATIVEX
```

Comment:
Is this the right logic? It looks like this will return ture if *any* of these faces are found, but we want to return true only if *all* of the faces are found.

Reply:
I think that's correct, this create a `Caps2` bitflag containing all directions (with each being a distinct bit), so if one is missing it will return false. Just checked to be sure:

```rust
    let all = Caps2::CUBEMAP_NEGATIVEX
        | Caps2::CUBEMAP_NEGATIVEY
        | Caps2::CUBEMAP_NEGATIVEZ
        | Caps2::CUBEMAP_POSITIVEX
        | Caps2::CUBEMAP_POSITIVEY
        | Caps2::CUBEMAP_POSITIVEZ;
    dbg!(Caps2::CUBEMAP_NEGATIVEX.contains(all));
    dbg!(all.contains(all));
```
returns `false` then `true`, as expected.

Reply:
Additionally, there should be a `CUBEMAP_ALLFACES` flag that OR all directions according to [this DDS doc](https://learn.microsoft.com/en-us/windows/win32/direct3ddds/dds-header), I'll see if I can add it to `ddsfile`.